### PR TITLE
Feat/check-item-existence

### PR DESCRIPTION
**Pull request by iglenwoo**
I have changed item checking api to use internal function instead of calling Gitlab API for better performance.
This may require more time to check multiple files at first.
However, it will eventually boost up the speed since it downloads items and uses the local items for the checking.

### CODE REVIEW

**Comment by iglenwoo** [comment]
@tnoelcke Please review this PR.

File: item-review-viewer/src/main/java/org/smarterbalanced/itemreviewviewer/web/services/GitLabService.java [hunk 1]
```diff
@@ -665,19 +666,15 @@ private void _downloadStim(String namespace, String bankKey, String id) {
     }
 
     @Override
```
Comment (tnoelcke) [inline item-review-viewer/src/main/java/org/smarterbalanced/itemreviewviewer/web/services/GitLabService.java:668]: Weren't we planning to have this function handle multiple items? Other wise we need to make a request per item on the front end which will be slow. I'd prefer to pass it the whole list of items and have it process them all at once.

**Review by tnoelcke — CHANGES_REQUESTED** [review]
I'd like to be able to process multiple items at a time. Thanks!

File: item-review-viewer/src/main/java/org/smarterbalanced/itemreviewviewer/web/services/GitLabService.java (hunk 1 above)
Comment (iglenwoo) [reply item-review-viewer/src/main/java/org/smarterbalanced/itemreviewviewer/web/services/GitLabService.java:668]: This function is used in the controller side which iteratively calls it.
So the logic of calling multiple items in RenderItemController.java file line: 258

**Review by tnoelcke — APPROVED** [review]

**Review by tnoelcke — APPROVED** [review]

**Pull request merged**